Mage/Module.cs

```diff
@@ -126,17 +160,12 @@ public void SetupUsercommands(Action<Exception> cb)
 			commandHandlerActions = new Dictionary<string, Action<JObject, Action<Exception, JToken>>>();
 			commandHandlerFuncs = new Dictionary<string, Func<JObject, UserCommandStatus>>();
 
-			if (Commands == null)
```

Comment:
Honestly this comes down to how we want the user to implement it, if we force the user to override it by using abstract any modules they want will need to have the override.

The question is how many modules do NOT have commands. Also do we ever create a module without commands to start with and then add later (this is more probable than a module not having commands).

I don't honestly mind either, we can use abstract but we have to be aware when we are developing setting this up with an empty list should be ok.